Neuron/Sections/Transaction/Send/SendTransactionViewController.swift

```diff
@@ -78,7 +87,7 @@ class SendTransactionViewController: UITableViewController, TransactonSender {
             let realm = try! Realm()
             let ether = realm.objects(TokenModel.self).first(where: { $0.type == .ether })!
             if ether.tokenBalance < paramBuilder.txFeeNatural {
-                Toast.showToast(text: "请确保账户剩余\(token.gasSymbol)高于矿工费用，以便顺利完成转账～")
+                Toast.showToast(text: "Transaction.Send.balanceNotSufficient".localized().replacingOccurrences(of: "[symbol]", with: token.gasSymbol))
```

Comment:
We need to double check if a token can use `[symbol]` as symbol name, although even if they can it's still very unlikely to happen. 😄